### PR TITLE
refactored severityvaluesgroup with react hooks

### PR DESCRIPTION
**Pull request by saberlynx**
**Checklist**:

- [] Tests
- [] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: gsa/src/web/components/powerfilter/severityvaluesgroup.js [hunk 1]
```diff
@@ -16,64 +16,41 @@
...
+
+const SeverityValuesGroup = ({filter, name, title, onChange}) => {
+  const [rel, setRel] = useState('=');
+  const severity = parseSeverity(filter.get(name));
+  const keyword = name;
+
+  return (
+    <FormGroup title={title}>
+      <Divider>
+        <RelationSelector relation={rel} onChange={newRel => setRel(newRel)} />
```
Comment (bjoernricks) [inline gsa/src/web/components/powerfilter/severityvaluesgroup.js]: I am seeing a potential bug here. Changing the relation isn't propagated to the calling component anymore. You should call the onChange handler here too. Something like
```jsx
<RelationSelector relation={rel} onChange={newRel => {setRel(newRel); onChange(severity, name, newRel);} />
```

File: gsa/src/web/components/powerfilter/severityvaluesgroup.js [hunk 2]
```diff
@@ -16,64 +16,41 @@
...
-          />
-        </Divider>
-      </FormGroup>
-    );
-  }
-}
+import Divider from 'web/components/layout/divider';
+
+const SeverityValuesGroup = ({filter, name, title, onChange}) => {
+  const [rel, setRel] = useState('=');
```
Comment (bjoernricks) [inline gsa/src/web/components/powerfilter/severityvaluesgroup.js]: I am thinking about another issue which has been in the component before already. Could it be possible that the relation is not "=" when initially being rendered. I guess we have to get the relation from the filter.

```js
const term = filter.getTerm(name);
severity = isDefined(term) ? parseSeverity(term.value) : undefined;
relation = isDefined(term) ? term.relation : '=';
...
```

Comment (saberlynx) [reply gsa/src/web/components/powerfilter/severityvaluesgroup.js]: Thanks for this catch. I put the relation bit into the useState argument.

File: gsa/src/web/components/powerfilter/severityvaluesgroup.js (hunk 1 above)
Comment (saberlynx) [reply gsa/src/web/components/powerfilter/severityvaluesgroup.js]: thanks. done.

**Pull request merged**